src/main/java/com/google/graphgeckos/dashboard/datatypes/Log.java

```diff
@@ -31,11 +31,16 @@
   /**
    * Constructs an instance of Log. Both type and link can be null.
    */
-  Log(String type, String link) {
+  public Log(String type, String link) {
     this.type = type;
     this.link = link;
   }
 
+  public Log(String[] logComponents) {
```

Comment:
When is this alternative constructor used?

Reply:
It is needed to unpack json properly.

Reply:
Cool! It might be worth adding a comment to the constructors to differentiate them.